modules/api/docs/LorisRESTAPI_v0.0.4-dev.md

```diff
@@ -1,5 +1,4 @@
-# Loris API - v0.0.2
```

Comment:
It looks like I modified a lot of things in the doc of v0.0.2, but it is actually modified from v0.0.3. I removed `LorisRESTAPI.md` and added `LorisRESTAPI_v0.0.4-dev.md`. 

I did not modifiy the original `LorisRESTAPI_v0.0.3.md` so Github must consider I replaced `v0.0.2` by `v0.0.4-dev`.

Reply:
I solved this problem by renaming `LorisRESTAPI_v0.0.3.md` to `LorisRESTAPI.md`.